client/src/store/index.ts

```diff
@@ -14,7 +14,7 @@ export const useMapSettings = () => {
     'map-settings',
     parseAsJson<{ basemap: string; labels: string | null }>().withDefault({
       basemap: 'basemap-satellite',
-      labels: 'light',
+      labels: null,
```

Comment:
Not sure if this is intended as we don't have it on the acceptance criteria but when we change of module (e.g. network to geospacial data) even if we have some labels active the labels are deactivated again. Shouldn't we preserve the user selection?

Reply:
This line is just the default value. The labels disappear when changing module because the query params are not kept in the URL.

I agree with you we should check with Lidia and Carla. Do you mind if we keep it like this for now and address this in another task and PR?